Dockerfile

```diff
@@ -1,6 +1,4 @@
-FROM golang:1.17.7-alpine3.15 as builder
-
-RUN apk add --update --no-cache ca-certificates tzdata git make bash && update-ca-certificates
+FROM registry.ci.openshift.org/openshift/release:golang-1.17 as builder
```

Comment:
we can add a ARG so the image can be build by other images, so if anyone wants to override the registry image, they have an alternative

Reply:
ARG has no way to pass a default option and passing the image name during the build process would mean that we have to modify the boilerplate :/

Reply:
I didn't know this. Thank you!

Reply:
I didn't know this. Thank you!

Reply:
@georgettica fixed